### DISCUSSION TITLE
How to export Chats from Mac OS app?

### DISCUSSION THREAD

**Discussion by m-r-m-s** [lencx/ChatGPT#287] [Q&A] [answered]
Hi, 

I am using the Mac OS application on 10.14.6 and I am not seeing an option to export previous Chat conversations. Is this functionality not implemented yet? On the Repo it says this functionality is available here: 
https://github.com/lencx/ChatGPT#-features

Thanks in advice for providing any insight on how to use.

**Comment by m-r-m-s** [comment, marked as answer]
I now see this is a bug that is being addressed: https://github.com/lencx/ChatGPT/issues/274#issuecomment-1407541060

**Reply by lencx** [reply]
Yes, because openai made changes to the site that prevented the previous script from working.

**Reply by m-r-m-s** [reply]
Hi, I downloaded the latest version and there is still no export option visible even when I resize the window for the application. I am on Mac OS 10.14.6. Also, I tried a complete uninstall and re-install but same issue. Also, I noticed that even though I completed deleted the application, my ChatGPT log in credentials still persisted? Besides deleting the application, I also deleted:
```
~/Library/Caches/com.lencx.chatgpt
~/Library/WebKit/com.lencx.chatgpt
```

**Reply by lencx** [reply]
None of these buttons exist?
<img width="400" alt="export" src="https://user-images.githubusercontent.com/16164244/216351054-34a5d373-eb8b-40c8-a648-ec20cf5b33d3.png">

**Comment by m-r-m-s** [comment]
Correct. This is what it looks like in previous chats and a new one i just generated:

![Screen Shot 2023-02-02 at 9 38 27 AM](https://user-images.githubusercontent.com/16727232/216354390-9de46a45-7150-4817-a22b-76ffbf8e4b7f.png)

**Reply by lencx** [reply]
What is the content of your configuration file `~/.chatgpt/chat.conf.json`?

**Reply by m-r-m-s** [reply]
```
{
  "titlebar": false,
  "hide_dock_icon": false,
  "theme": "Light",
  "auto_update": "Prompt",
  "tray": true,
  "popup_search": false,
  "stay_on_top": false,
  "main_dashboard": false,
  "tray_dashboard": false,
  "main_origin": "https://chat.openai.com",
  "tray_origin": "https://chat.openai.com",
  "default_origin": "https://chat.openai.com",
  "ua_window": "",
  "ua_tray": "Mozilla/5.0 (iPhone; CPU iPhone OS 16_0 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.0 Mobile/15E148 Safari/604.1",
  "global_shortcut": ""
}
```

**Reply by lencx** [reply]
If you're sure you're installing the latest version, I can't find a reason for it at the moment, as the configuration appears to be normal.

**Reply by m-r-m-s** [reply]
ok, thanks for checking. I am installing latest version from Repo releases (DMG version). Could it have to do with ChatGPT being over capacity?

Also, can you explain why the log-in credentials persist even after I have completely deleted the installation?
I also deleted` ~/.chatgpt/chat.conf.json` in addition to above mentioned files and log in still persists until I actually "log out"
